docs/source/development.rst

```diff
@@ -0,0 +1,75 @@
+Developer Information
+=====================
+
+* Please read the `CONTRIBUTING`_ guidelines.
+* You probably want to `install from source`_.
+
+Release Engineering
+-------------------
+
+Pre-release
+^^^^^^^^^^^
+
+* Update version in `docs/source/conf.py`.
+* Edit the `CHANGELOG`_.
+* Ensure tox tests pass.
+
+Release
+^^^^^^^
+
+Tag the release and push to github.com
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+  .. code-block:: bash
+
+      $ git tag -a 1.0.5 -m "Version 1.0.5"
+      $ git push origin 1.0.5
```

Comment:
We are using `$` for this project.  All of the existing documentation follows this pattern.